### PR TITLE
feat: added col breaks feature

### PR DESCRIPTION
**Pull request by StefanNedelchev**
## Summary

The library supports row breaks but doesn't support col breaks which is also a valid XLSX feature.

## Test plan

Added unit test and integration test for writing col breaks

```shell
Page Breaks
      ✓ adds multiple row breaks
      ✓ adds multiple col breaks
```

### CODE REVIEW

**Comment by StefanNedelchev** [comment]
I think I'll need a help with this because although the library now correctly writes col breaks, I think it doesn't properly read them, so if the document has any, they will be lost. Apparently this needs to be covered with tests which will make it easier to detect and fix

File: package.json [hunk 1]
```diff
@@ -111,7 +111,7 @@
     "saxes": "^5.0.1",
     "tmp": "^0.2.0",
     "unzipper": "^0.10.11",
-    "uuid": "^8.3.0"
+    "uuid": "^9.0.0"
```
Comment (StefanNedelchev) [inline package.json:114]: I remember last year uuid released v9 to address vulnerability issue so it's always good to keep up to date with it

**Comment by zurmokeeper** [comment]
@StefanNedelchev That's really great, thanks for the PR, but I'm going to have to spend some time getting to grips with this PR first, will reply as soon as I make progress!

File: lib/xlsx/xform/sheet/col-breaks-xform.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const PageBreaksXform = require('./page-breaks-xform');
+
+const ListXform = require('../list-xform');
+
+class ColBreaksXform extends ListXform {
```
Comment (StefanNedelchev) [inline lib/xlsx/xform/sheet/col-breaks-xform.js:7]: This class has been using `RowBreaksXform` as an example

File: lib/doc/column.js [hunk 3]
```diff
@@ -201,6 +199,21 @@ class Column {
     });
   }
 
+  // Page Breaks
+  addPageBreak(t, b) {
```
Comment (StefanNedelchev) [inline lib/doc/column.js:203]: The function was copied from `row.js` and modified to work for col break

**Comment by StefanNedelchev** [comment]
> @StefanNedelchev That's really great, thanks for the PR, but I'm going to have to spend some time getting to grips with this PR first, will reply as soon as I make progress!

No worries, take your time. I added some refinements and also added some comments for better explanation

**Review by zurmokeeper — APPROVED** [review]

**Comment by zurmokeeper** [comment]
@StefanNedelchev Thank you very much for your contribution, it's been merged and a new version has been released.

You are welcome to continue to PR and improve the package together, I see that there are still a lot of issues and good PRs in exceljs, my plan is to finish the issues in the current project and then go to exceljs to see what else can be solved! Thanks again!

**Comment by StefanNedelchev** [comment]
> @StefanNedelchev Thank you very much for your contribution, it's been merged and a new version has been released.
> 
> You are welcome to continue to PR and improve the package together, I see that there are still a lot of issues and good PRs in exceljs, my plan is to finish the issues in the current project and then go to exceljs to see what else can be solved! Thanks again!

Thanks, it was my pleasure. It's sad to see a decent free stylesheet library to get left behind so I'm happy to contribute to this revived project. Since the PR is merged I'll delete my branch and my fork (until next PR)

**Comment by zurmokeeper** [comment]
Yes, you're right, it was my problem, I changed it in the next update.

**Pull request merged**